types/api/txAction.ts

```diff
@@ -21,6 +21,32 @@ export interface TxActionNft {
   };
 }
 
-export type TxAction = {
+export type TxGolemBaseAction = {
+  type: 'golembase_entity_created' | 'golembase_entity_updated';
+  data: {
+    entity_id: string;
+    expiration_block: number;
+  };
+} | {
+  type: 'golembase_entity_deleted';
+  data: {
+    entity_id: string,
+  };
+} | {
+  type: 'golembase_entity_ttl_extended';
+  data: {
+    entity_id: string,
+    old_expiration_block: number,
+    new_expiration_block: number,
+  }
+}
```

Comment:
Please refactor the sub-types into their own named types, making it more readable that:
`TxGolemBaseAction` is an union of `TxGolemBaseEntityCreated`, `TxGolemBaseEntityUpdated`, `TxGolemBaseEntityDeleted` and `TxGolemBaseEntityTTLExtended`.